recipe/build.sh

```diff
@@ -14,13 +14,19 @@ fi
 
 for _shared in OFF ON; do
   mkdir build-${_shared}
-  pushd build-${_shared}
+  pushd build-${_shared} || exit
     cmake ${CMAKE_ARGS}                     \
           -DCMAKE_INSTALL_PREFIX=${PREFIX}  \
           -DBUILD_SHARED_LIBS=${_shared}    \
```

Comment:
Right now it first builds a static lib, and then it builds both static+shared. I would remove the loop and either we build static+shared in one go or we remove the static lib entirely.

Reply:
We should remove the static lib entirely. If it becomes needed at some point, we'll had a libssh2-static output.

Reply:
I removed the static lib entirely